### ISSUE TITLE
Bug: Zero-Unit insists that flex-shrink: 0 have a unit applied

### ISSUE THREAD

**Issue by TheJaredWilcurt** [sasstools/sass-lint#1236]
**Sass Lint Version:** 1.12.1
**Relevant parts of configuration:**
```yaml
rules:
  zero-unit:
    - 1
    - include: true
```
**Source code**:
```sass
.example
    flex-shrink: 0
```
**Expectation:** Lint should pass
**Actual:** `Unit required for values of 0  │ zero-unit`

**Comment by TheJaredWilcurt** [comment]
Note that `flex-shrink` is meant to be unitless.

* https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink
